project/Release.scala

```diff
@@ -29,7 +29,7 @@ object Release {
     val repo = extracted.get(Publish.defaultPublishTo)
     val state1 = extracted.runAggregated(publishSigned in projectRef, state)
     val (state2, Seq(api, japi)) = extracted.runTask(unidoc in Compile, state1)
-    val (state3, docs) = extracted.runTask(generate in Sphinx, state2)
+    val (state3, docs) = extracted.runTask(paradox in ProjectRef(projectRef.build, "akka-docs") in Compile, state2)
 
```

Comment:
As far as I understood the 'paradox' task here will put the target location of the paradox docs into `docs`, and the code below will copy it into `target/release/docs/akka/<version>`. this `target/release` folder then gets rsync'ed - so I think this should be fine (though indeed I haven't tested)